### PR TITLE
Feature api

### PR DESCRIPTION
**Pull request by DVA09**

### CODE REVIEW

File: API/BDD/features/api.feature [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Feature: API testing different endpoints
+
+  Scenario Outline: Create a new user
+
+    Given Send request to "https://petstore.swagger.io/v2/user"
+    Then Check status code, type, message of responce
+
+  Scenario Outline: Create a new user negative
```
Comment (Alex19777) [inline API/BDD/features/api.feature]: Do you need Outline here?

File: API/BDD/features/api.feature [hunk 2]
```diff
@@ -0,0 +1,31 @@
+Feature: API testing different endpoints
+
+  Scenario Outline: Create a new user
+
```
Comment (Alex19777) [inline API/BDD/features/api.feature:9]: Why do you need empty line here?

File: API/BDD/features/api.feature (hunk 1 above)
Comment (DVA09) [reply API/BDD/features/api.feature]: Deleted Outline from Scenario

File: API/BDD/features/step-definitions/steps.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        "email": "testuser@example.com",
+        "password": "test12356_)",
+        "phone": "+375297891234",
+        "userStatus": 0
+    };
+    response = await axios.post(url, user);
+});
+
+Then(/^Check status code, type, message of responce negative$/, async () => {
+	await expect(response.status).toBe(200);
```
Comment (Alex19777) [inline API/BDD/features/step-definitions/steps.js]: Fix spaces

File: API/BDD/features/step-definitions/steps.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    });
+});
+
+Then(/^Check status code, type of DELETE responce$/, async () => {
+	await expect(response.status).toBe(200);
+    await expect(response.data.type).toBe("unknown");
+    await expect(response.data.message).toBe("New User");
+});
+
+
```
Comment (Alex19777) [inline API/BDD/features/step-definitions/steps.js]: Fix spaces

File: API/BDD/features/api.feature (hunk 2 above)
Comment (DVA09) [reply API/BDD/features/api.feature:9]: It was in the template

File: API/BDD/wdio.conf.js [hunk 5]
```diff
@@ -0,0 +1,333 @@
+exports.config = {
+    //
+    // ====================
+    // Runner Configuration
+    // ====================
+    // WebdriverIO supports running e2e tests as well as unit and component tests.
```
Comment (Alex19777) [inline API/BDD/wdio.conf.js]: looks like autogenerated file

File: API/BDD/features/api.feature [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    Then Check status code, type, message of responce
+
+  Scenario Outline: Create a new user negative
+
+    Given Send negative POST request to "https://petstore.swagger.io/v2/user"
+    Then Check status code, type, message of responce negative 
+
+  Scenario Outline: User with ID 0 exists
+
+    Given Send GET request to "https://petstore.swagger.io/v2/user/login?username=Test%20UserLastName&password=test12356_)"
```
Comment (Alex19777) [inline API/BDD/features/api.feature]: Move common part of URL to some constant

File: API/BDD/features/api.feature [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    Then Check status code, type of GET responce
+  
+  Scenario Outline: Updated user
+
+    Given Send PUT request to "https://petstore.swagger.io/v2/user/New%20User"
+    Then Check status code, type, message of PUT responce
```
Comment (Alex19777) [inline API/BDD/features/api.feature]: It is better to use When -> Then steps

File: API/BDD/features/step-definitions/steps.js (hunk 3 above)
Comment (DVA09) [reply API/BDD/features/step-definitions/steps.js]: Fixed it

File: API/BDD/features/step-definitions/steps.js (hunk 4 above)
Comment (DVA09) [reply API/BDD/features/step-definitions/steps.js]: Fixed it

File: API/BDD/features/step-definitions/steps.js [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+Given(/^Send request to "([^"]*)"$/, async (url) => {
+	const user = {
+        "id": 22,
+        "username": "Test UserLastName",
+        "firstName": "Test",
+        "lastName": "UserLastName",
+        "email": "testuser@example.com",
+        "password": "test12356_)",
+        "phone": "+375297891234",
+        "userStatus": 0
```
Comment (Alex19777) [inline API/BDD/features/step-definitions/steps.js]: Make object once and use when need it multiple times

File: API/BDD/wdio.conf.js (hunk 5 above)
Comment (DVA09) [reply API/BDD/wdio.conf.js]: Deleted this file

File: API/BDD/features/api.feature (hunk 7 above)
Comment (DVA09) [reply API/BDD/features/api.feature]: Fixed it

File: API/BDD/features/step-definitions/steps.js (hunk 8 above)
Comment (DVA09) [reply API/BDD/features/step-definitions/steps.js]: Declared the object at the beginning

File: API/BDD/features/api.feature (hunk 6 above)
Comment (DVA09) [reply API/BDD/features/api.feature]: Done it

File: API/BDD/features/api.feature (hunk 2 above)
Comment (Alex19777) [reply API/BDD/features/api.feature:9]: Remove this lines

Comment (DVA09) [reply API/BDD/features/api.feature:9]: Deleted it

File: API/BDD/features/step-definitions/steps.js [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+    "phone": "+375297891234",
+    "userStatus": 0
+};
+
+Given(/^Send positive POST request$/, async () => {    
+    response = await axios.post(address.url, user);
+});
+
+When(/^Check status code, type, message of responce$/, async () => {
+	await expect(response.status).toBe(200);
```
Comment (Alex19777) [inline API/BDD/features/step-definitions/steps.js]: Remove redundant spaces here and in other places

Comment (DVA09) [reply API/BDD/features/step-definitions/steps.js]: Fixed it

**Review by Alex19777 — APPROVED** [review]

**Pull request merged**